cms/djangoapps/contentstore/config/waffle_utils.py

```diff
@@ -0,0 +1,13 @@
+"""Util methods for Waffle checks"""
+
+from cms.djangoapps.contentstore.config.waffle import ENABLE_CHECKLISTS_QUALITY
+
+
+def should_show_checklists_quality(course_key):
```

Comment:
Yes you are right. I have changed the flag to be True be default i.e if a flag for a course does not exist it would considered to be enabled by default. For the courses we want to disable the course quality we can simply add the flag and set it to False.